src/org/mozilla/javascript/NativeString.java

```diff
@@ -231,6 +233,18 @@ public Object execIdCall(IdFunctionObject f, Context cx, Scriptable scope,
                     continue again;
                 }
 
+                case ConstructorId_fromCodePoint: {
+                    int N = args.length;
```

Comment:
please let us follow the java code style and start variable names with lowercase char

Reply:
Oh, I just keep the style as exists code, is there a schema for formatter?